source/opcode.h

```diff
@@ -31,166 +31,84 @@
 #include "libspirv/libspirv.h"
 #include "table.h"
 
-// Functions
-
-/// @brief Get the name of the SPIR-V generator.
-///
-/// See the registry at
-/// https://www.khronos.org/registry/spir-v/api/spir-v.xml
-///
-/// @param[in] generator Khronos SPIR-V generator ID
-///
-/// @return string name
+// Gets the name of the SPIR-V generator.
+//
+// See the registry at https://www.khronos.org/registry/spir-v/api/spir-v.xml.
 const char* spvGeneratorStr(uint32_t generator);
 
-/// @brief Combine word count and Opcode enumerant in single word
-///
-/// @param[in] wordCount Opcode consumes
-/// @param[in] opcode enumerant value
-///
-/// @return Opcode word
-uint32_t spvOpcodeMake(uint16_t wordCount, SpvOp opcode);
-
-/// @brief Split the binary opcode into its constituent parts
-///
-/// @param[in] word binary opcode to split
-/// @param[out] wordCount the returned number of words (optional)
-/// @param[out] opcode the returned opcode enumerant (optional)
+// Combines word_count and opcode enumerant in single word.
+uint32_t spvOpcodeMake(uint16_t word_count, SpvOp opcode);
+
+// Splits word into into two constituent parts: word_count and opcode.
 void spvOpcodeSplit(const uint32_t word, uint16_t* wordCount, SpvOp* opcode);
 
-/// @brief Find the named Opcode in the table
-///
-/// @param[in] table to lookup
-/// @param[in] name name of Opcode to find
-/// @param[out] pEntry returned Opcode table entry
-///
-/// @return result code
+// Finds the named opcode in the given opcode table. On success, returns
+// SPV_SUCCESS and writes a pointer of the table entry into *entry.
```

Comment:
`spv_opcode_desc` is a `typedef` of `spv_opcode_desc_t*`. But yeah, may be `a handle to the table entry` is better?